README.md

```diff
@@ -1,3 +1,39 @@
 # API-Monitor
 [![CircleCI](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop.svg?style=svg)](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop)
 [![Coverage Status](https://coveralls.io/repos/github/Ashaba/API-Monitor/badge.svg?branch=develop)](https://coveralls.io/github/Ashaba/API-Monitor?branch=develop)
+ 
+ An application the performs health checks of an api
```

Comment:
An application that ...

Reply:
You missed out the above change request in your updated commit

Reply:
It's just a statement, doesn't need to be a heading. I didn't miss it

Reply:
I totally agree. What I'm saying is that the statement should read as *An application **that** ...* and not *An application **the** ...*